extensions/Khronos/KHR_mesh_compression/README.md

```diff
@@ -0,0 +1,172 @@
+# KHR_draco_geometry_compression
```

Comment:
I changed the name back to *_mesh_*. And removed `geometryType` in extension. Added description for restriction in README.